marketplace_builder/views/pages/tutorials/variables/context.liquid

```diff
@@ -0,0 +1,112 @@
+---
+converter: markdown
+metadata:
+  title: The context variable
+  description: This guide will walk you thorugh the details of the context variable
+slug: tutorials/variables/context
+searchable: true
+---
+
+{{ page.metadata.description }}
+
+## Requirements
+
+To follow the steps in this tutorial, you should be familiar with:
+ * [Liquid Template language](https://shopify.github.io/liquid/)
+
+## Steps
+
+To learn everything about `context` variable please follow this three easy steps:
+1. {% assign step_1 = "What is context variable and how to use it." %}[{{ step_1 }}](#step-1)
+2. {% assign step_2 = "Context variable components" %}[{{ step_2 }}](#step-2)
+3. {% assign step_3 = "Expose local variable within context variable" %}[{{ step_3 }}](#step-3)
+
+### <a name="step-1" />Step 1: {{ step_1 }}
```

Comment:
What is this?

Reply:
Currently, we don't use linking to steps. This is planned for later universally, if needed. So for now, to remain consistent, we don't add links.

Reply:
Sidenote: `a name` doesnt link to anything - `a href` is for linking, with #id of element if its internal
 to the page.